op-batcher/batcher/driver.go

```diff
@@ -286,84 +287,114 @@ func (l *BatchSubmitter) loop() {
 
 	ticker := time.NewTicker(l.PollInterval)
 	defer ticker.Stop()
+
+	receiptsCh := make(chan txmgr.TxReceipt[txData])
```

Comment:
hi, is it better to use buffered channel?

Reply:
I also thought about that, but I think it's not necessary here. It changes the behavior of `Queue.Wait`. Right now, it is guaranteed to only return after the last receipt has been read from the channel, because it's non-buffered. If we add a buffer here, `Queue.Wait` would return earlier. It depends on what behavior we want `Wait` to have. And if we want buffered behavior instead, the buffer needs to be large enough, or behavior might be unpredicted.